pages/builds/controlling_concurrency.md.erb

```diff
@@ -2,27 +2,61 @@
 
 Some tasks need to be run with very strict concurrency rules to ensure they don’t collide with each other. Common examples for needing concurrency control are deployments, app releases and infrastructure tasks.
 
-To help you control concurrency Buildkite provides two primitives: concurrency groups, and concurrency limits.
+To help you control concurrency Buildkite provides two primitives: concurrency limits, and concurrency groups. While these two primitives are interdependant, they are not explicitly linked.
 
 <%= toc %>
 
+## Concurrency Limits
+
+Concurrency limits define the number of jobs that are allowed to be running at any one time. These limits are set per-step and only apply to jobs that are based on that step. 
+
+```yml
+steps:
+- command: "tests.sh"
+  concurrency: 1
```

Comment:
This example will blow up with the "Missing concurrency_group_id for ..." error, maybe it needs a concurrency group?